npdarrington/fitlit
# Nd design ui

What’s this PR do?  
- This PR is the re-design of the intended UI after getting the data model to display correctly.
- A full design change featuring leaderboard, individual data cards for current user and links on the left that will change what data is being displayed.
- Icons have been included.
 
Where should the reviewer start?  
- Git pull this branch for newest changes.
 
How should this be manually tested?  
- Open index.html.
- Check for styling on the screen and how it displays.
- Ensure color scheme is to our liking.
 
Any background context you want to provide?  
- This is intended for full size laptop and desktop screens only.
- Graphs will be added in a future iteration.
- Calendar will be implemented in a future iteration.
- The data will have to be hooked back up. This is only the HTML/CSS portion of the new UI update.
 
What are the relevant tickets?  
Closes #61 
 
Screenshots (if appropriate)  
- N/A
 
Questions: 
- N/A
